client/my-sites/plugins/plugins-category-results-page/header.tsx

```diff
@@ -0,0 +1,21 @@
+import './header.scss';
+
+const Header = ( {
+	title,
+	subtitle,
+	count,
+}: {
+	title: string;
+	subtitle: string;
+	count: string;
+} ) => {
+	return (
+		<div className="plugin-category-results-header">
+			<h1 className="plugin-category-results-header__title">{ title }</h1>
+			<p className="plugin-category-results-header__count">{ count }</p>
+			{ subtitle && <h2 className="plugin-category-results-header__subtitle">{ subtitle }</h2> }
```

Comment:
Could we flip these around? @SaxonF any complaints?

![Screenshot(3)](https://user-images.githubusercontent.com/811776/185541121-77c2282d-838b-4946-81b6-d6abd310d198.png)

----

e.g. something like this:
![Screenshot 2022-08-19 at 14-13-14 Plugins ‹ Business - Simple Site — WordPress com](https://user-images.githubusercontent.com/811776/185541443-f9ca27e8-a47e-4951-94df-c8bbce0786eb.png)

Tying this into another couple of changes we're discussing in p1660878739236139/1660860939.096259-slack-CKZHG0QCR @Rolilink @lupus2k

Reply:
Will move the plugins down in this PR.